ksql-common/src/main/java/io/confluent/ksql/schema/ksql/LogicalSchema.java

```diff
@@ -62,6 +66,16 @@
       SchemaUtil.ROWKEY_NAME
   );
 
+  private static final Schema IMPLICIT_SCHEMA = SchemaBuilder
+      .struct()
+      .field(SchemaUtil.ROWTIME_NAME, Schema.OPTIONAL_INT64_SCHEMA)
+      .build();
+
+  private static final Schema KEY_SCHEMA = SchemaBuilder
+      .struct()
```

Comment:
just some musings that I think you can ignore: I feel like this (and `IMPLICIT_SCHEMA`) makes more sense as `List<Field>` because it is often that the key schema itself is not a single field wrapped with a struct but rather just a single field.

Reply:
`IMPLICIT_SCHEMA` will go very shortly.  

As for struct vs fields, sure, in time my plan is for this class to be a list of implicit, key and value fields, where the field type is our own, but what's there now keeps the implementation simple while we're working with Connect schemas, which likes to deal with whole schemas, not lists of fields.